src/schema/capital-commitment.ts

```diff
@@ -16,14 +16,14 @@ export const capitalCommitment = pgTable(
   "capital_commitment",
   {
     id: uuid("id").primaryKey(),
-    type: char("type", { length: 4 }).references(
-      () => capitalCommitmentType.code,
-    ),
-    plannedDate: date("planned_date"),
+    type: char("type", { length: 4 })
```

Comment:
These changes should also be captured in the generated migration file

Reply:
doh!! 
i was leaving that for last and forgot.